spec/features/issues_spec.rb

```diff
@@ -134,15 +134,15 @@
     it 'sorts by newest' do
       visit project_issues_path(project, sort: sort_value_recently_created)
 
-      first_issue.should include("foo")
-      last_issue.should include("baz")
+      expect(first_issue).to include("foo")
+      expect(last_issue).to include("baz")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.